app/auth/authenticate.js

```diff
@@ -24,6 +25,7 @@ const authenticate = async (request) => {
   session.setToken(request, sessionKeys.tokens.tokenExpiry, expiresIn.toISOString(redeemResponse.expires_in))
   session.setCustomer(request, sessionKeys.customer.crn, accessToken.contactId)
   session.setCustomer(request, sessionKeys.customer.organisationId, accessToken.currentRelationshipId)
+  session.setCustomer(request, sessionKeys.customer.attachedToMultipleBusinesses, accessToken.enrolmentCount && accessToken.enrolmentCount > 1)
```

Comment:
session.setCustomer(request, sessionKeys.customer.attachedToMultipleBusinesses, typeof accessToken.enrolmentCount !== 'undefined' && accessToken.enrolmentCount > 1)
--
if we want the value to be true/false
otherwise sessionKeys.customer.attachedToMultipleBusinesses will be either 'undefined' or true

Reply:
good point, fixed!